study.md

```diff
@@ -112,13 +116,15 @@ From a collection of galaxies, update the `name` attribute of the record with
 the `designation` attribute of "NGC 224" to "Andromeda".
 
 ```ruby
-# your answer here
+g = Galaxy.find_by(designation: 'NGC 224')
+g.update(name: 'Andromdea')
 ```
 
 ## Delete
 
 From a collection of characters, delete the record with the `id` attribute of 4.
 
 ```ruby
-# your answer here
+c = Character.find_by(id: 14)
+c.destroy
 ```
```

Comment:
Correct! when finding by ids we can also just use `Character.find(14)` for short hand